packages/interactivity-router/src/assets/test/styles.test.ts

```diff
@@ -0,0 +1,561 @@
+/**
+ * Internal dependencies
+ */
+import {
+	updateStylesWithSCS,
+	prepareStyles,
+	applyStyles,
+	type StyleElement,
+} from '../styles';
+
+const mockSheet = (
+	element: StyleElement,
+	{ disabled, mediaText }: { disabled: boolean; mediaText: string }
+) => {
+	if ( element.sheet ) {
+		Object.assign( element.sheet, { disabled, media: { mediaText } } );
+	} else {
+		Object.defineProperty( element, 'sheet', {
+			value: {
+				disabled,
+				media: { mediaText },
+			},
+		} );
+	}
+};
+
+// Mock DOM elements for testing
```

Comment:
You are missing a few ending periods in the comments of this file. They could also use third person verbs.

```suggestion
// Mocks DOM elements for testing.
```

Reply:
The dots are fixed in https://github.com/woocommerce/gutenberg/pull/2/commits/519b45f10a28389ef124fceb336da50ed25ebf8e.

Regarding the usage of third-person verbs, I decided to keep them in the infinitive, as that's how most of the verbs in comments are written.

I've also added TSDocs for the mock functions in https://github.com/woocommerce/gutenberg/pull/2/commits/52b771317468489490000b6f1d787f7357baea2a.

Reply:
The standard in WordPress is to [use the third person](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#documenting-tips) in the comments.

Reply:
Well, then we'll have to update lots of TSDocs and comments. 😅 I'll create a PR for that later on.

Reply:
It is true that the comments of the Interactivity API are not very up-to-date to follow the WordPress standards 🙂

Maybe a task for AI?

Reply:
I would say so. 🤖

Reply:
I opened a PR. https://github.com/woocommerce/gutenberg/pull/11

Reply:
Thanks!